truststore_java.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"bytes"
+	"crypto/sha1"
+	"crypto/sha256"
+	"crypto/x509"
+	"encoding/hex"
+	"encoding/pem"
+	"fmt"
+	"hash"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+)
+
+var (
+	hasJava    bool
+	hasKeytool bool
+
+	javaHome    string
+	cacertsPath string
+	keytoolPath string
+	storePass   string = "changeit"
+)
+
+func init() {
+	if v := os.Getenv("JAVA_HOME"); v != "" {
```

Comment:
How common/expected is it for `JAVA_HOME` to be set in the default environment?

Reply:
On servers (as the user running java apps) it should exist. Developers _should_ have JAVA_HOME set, but they might not.

It's probably good enough for the 80% usecase.